File: IronVault/IronVault.Model/Models/BiznisReport.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class BiznisReport
    {

        public int UkupnoKorisnici { get; set; }
        public int KorisniciUTeretani { get; set; }
        public float UkupnaZaradaMjeseca { get; set; }
        public float ZaradaMjesecaNaClanarinama { get; set; }
        public float ZaradaMjesecaNaKupovinama { get; set; }

        public Suplement? NajprodavanijiSuplement { get; set; }
        public int? NajprodavanijiSuplementCount { get; set; }

        public Suplement? NajboljaOcjenaSuplement { get; set; }

        public Korisnik? NajaktivnijiKorisnik { get; set; }


    }
}

File: IronVault/UI/ironvault_desktop/lib/models/biznis_report.dart
import 'dart:ffi';

import 'package:ironvault_desktop/models/korisnik.dart';
import 'package:ironvault_desktop/models/suplement.dart';
import 'package:json_annotation/json_annotation.dart';

part 'biznis_report.g.dart';

@JsonSerializable()
class BizniReport {
  int? ukupnoKorisnici;
  int? korisniciUTeretani;
  double? ukupnaZaradaMjeseca;
  double? zaradaMjesecaNaClanarinama;
  double? zaradaMjesecaNaKupovinama;
  Suplement? najprodavanijiSuplement;
  int? najprodavanijiSuplementCount;
  Suplement? najboljaOcjenaSuplement;
  Korisnik? najaktivnijiKorisnik;

  BizniReport();

  factory BizniReport.fromJson(Map<String, dynamic> json) =>
      _$BizniReportFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$BizniReportToJson(this);
}
